app/js/components/patientDashboard/styles.scss

```diff
@@ -0,0 +1,64 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+ 
+.patient-header {
+  border-radius: 5px 5px 0px 0px;
+  margin: 10px 0px 0px 0px;
+  
+  .rotate180::before {
+    transform: rotate(180deg);
+  }
+  
+  .preformatted-note {
+    margin: 1px;
+    padding: 0px;
+    padding-top: 1px;
+    background: white;
+    color: #e9241d;
+    font-family: "OpenSans", Arial, sans-serif;
```

Comment:
this is been done twice in this file, could you somehow hold it in variable and reuse ?

Reply:
The `patentDashboard` component has been removed from the codebase consequently, its styles are also removed.